build/gocommand.js

```diff
@@ -76,6 +81,37 @@ function checkGo() {
 }
 
 /**
+ * Checks if go version fulfills the minimum version prerequisite, promises an error otherwise.
+ * @return {Q.Promise} A promise object.
+ */
+function checkGoVersion() {
+  let deferred = q.defer();
+  child.exec(
+      'go version',
+      {
+        env: env,
+      },
+      function(error, stdout) {
+        let match = /[\d\.]+/.exec(stdout.toString());  // matches version number
```

Comment:
Use https://github.com/npm/node-semver to compare versions :) It'll avoid a lot of troubles. I.e., I think current implementation does string comparison and is wrong when comparing e.g., `1.11.0` and `1.5.0`.